test/unit/style/style.test.js

```diff
@@ -360,67 +384,6 @@ test('Style#update', (t) => {
     });
 });
 
-test('Style#_resolve', (t) => {
-    t.test('creates StyleLayers', (t) => {
-        const style = new Style(new StubMap());
-        style.loadJSON({
-            "version": 8,
-            "sources": {
-                "foo": {
-                    "type": "vector"
-                }
-            },
-            "layers": [{
-                "id": "fill",
-                "source": "foo",
-                "source-layer": "source-layer",
-                "type": "fill"
-            }]
-        });
-
-        style.on('error', (error) => { t.error(error); });
-
-        style.on('style.load', () => {
-            t.ok(style.getLayer('fill') instanceof StyleLayer);
-            t.end();
-        });
-    });
-
-    t.test('handles ref layer preceding referent', (t) => {
```

Comment:
What was that test about, is it OK to kill?

Reply:
It's tested here: https://github.com/mapbox/mapbox-gl-js/blob/b9e10b939c6a3fe5d7ecac209f751b4871970ede/test/unit/style-spec/deref.test.js#L29